book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :author, :rentals
+end
```

Comment:
- [ ] You seem to have misunderstood what the previous reviewer was pointing out. You should have the previous method you had earlier. Just remove the `@rentals = ` and arrange your parameters.

  ```
  def add_rental(date, person)
    Rental.new(date, self, person)
  end
  ```

Reply:
Thank you @julie-ify